src/main/java/duke/gui/DialogueBox.java

```diff
@@ -0,0 +1,35 @@
+package duke.gui;
+
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
+public class DialogueBox extends HBox {
+    private Label text;
+    private ImageView displayPicture;
+
+    /**
+     * Constructor for a dialogue box.
+     *
+     * @param l  Text to be displayed.
+     * @param iv Window to contain text.
+     */
+    public DialogueBox(Label l, ImageView iv, boolean isUser) {
```

Comment:
Would using more descriptive names for `Label` and `ImageView` be preferred? 
```suggestion
    public DialogueBox(Label message, ImageView picture, boolean isUser) {
```